app/jobs/event_notification_job.rb

```diff
@@ -0,0 +1,6 @@
+class EventNotificationJob < ApplicationJob
+  helper EmailHelper
```

Comment:
Unneeded helper statement. The EmailHelper has no public api, so declaring it as a helper here does nothing